bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java

```diff
@@ -76,7 +77,28 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
                 return;
             }
 
+            // Change the CSR link relationship from `csr for` to `relates to`
+            // so that the CSRBot won't add the `csr` label automatically.
```

Comment:
I'm not sure this is a good idea. The only reason to keep the /csr command around would be to have a way of manually correcting the state of a PR if the bots somehow fail, but I'm leaning towards just removing the command. If a CSR is already linked in JBS, but is later deemed to not be needed, I think the user should fix that in JBS rather than through a Skara command.

I think this needs some discussion and input from more people.

Reply:
I read up more on the history here. The /csr command can be invoked prior to creating a CSR issue in JBS, so it still makes sense to have the command. Given this, then invoking "/csr unneeded" can happen in two different scenarios:

1. No CSR has been filed yet, then just remove the csr label
2. A CSR is linked in JBS, in this case the CSRBot will likely automatically add the label again

I still don't think it makes sense to have this command go and change things in JBS. In case 2, a better cause of action would be to tell the user in an error reply that a CSR already exists and needs to be closed/removed first.

Reply:
I revised the logic of the command `/csr unneeded` according to your comments below.